src/components/button/button.tsx

```diff
@@ -19,55 +19,154 @@ export class Button {
+            <ion-button color="success">Success</ion-button>
+            <ion-button color="warning">Warning</ion-button>
+            <ion-button color="danger">Danger</ion-button>
+            <ion-button color="secondary">Secondary</ion-button>
+            <ion-button color="tertiary">Tertiary</ion-button>
+            <ion-button color="light">Light</ion-button>
+            <ion-button color="medium">Medium</ion-button>
+            <ion-button color="dark">Dark</ion-button>
+          </div>
         </section>
 
         <section>
           <ion-list-header>
-            <ion-label>
-              Default Size
-            </ion-label>
+            <ion-label>Sizes</ion-label>
           </ion-list-header>
-          <ion-button color="success">Success</ion-button>
-          <ion-button color="warning">Warning</ion-button>
-          <ion-button color="danger">Danger</ion-button>
+          <div class="content">
+            <ion-button size="small">Small</ion-button>
+            <ion-button>Default</ion-button>
+            <ion-button size="large">Large</ion-button>
+            <ion-button size="xlarge">XLarge</ion-button>
+          </div>
         </section>
 
         <section>
           <ion-list-header>
-            <ion-label>
-              Large Size
-            </ion-label>
+            <ion-label>Shapes</ion-label>
           </ion-list-header>
-          <ion-button size="large" color="light">Light</ion-button>
-          <ion-button size="large" color="medium">Medium</ion-button>
-          <ion-button size="large" color="dark">Dark</ion-button>
+          <div class="content">
+            <ion-button>Default</ion-button>
+            <ion-button shape="round">Round</ion-button>
+            <ion-button shape="rectangular">Rectangular</ion-button>
+            <ion-button shape="round">
+              <ion-icon slot="icon-only" icon="heart"></ion-icon>
+            </ion-button>
+          </div>
         </section>
 
         <section>
           <ion-list-header>
-            <ion-label>
-              Block Width
-            </ion-label>
+            <ion-label>Fills</ion-label>
           </ion-list-header>
-          <ion-button class="ion-text-wrap" expand="block">A block button</ion-button>
+          <div class="content">
+            <ion-button>Default</ion-button>
+            <ion-button fill="outline">Outline</ion-button>
+            <ion-button fill="solid">Solid</ion-button>
+            <ion-button fill="clear">Clear</ion-button>
+          </div>
         </section>
 
         <section>
           <ion-list-header>
-            <ion-label>
-              Full Width
-            </ion-label>
+            <ion-label>States - Activated</ion-label>
           </ion-list-header>
-          <ion-button class="ion-text-wrap" expand="full" color="secondary">A full-width button</ion-button>
+          <div class="content">
+            <ion-button theme="ionic" class="ion-activated">
```

Comment:
Should these new buttons have the `ionic` theme hard-coded, even if you don't have the theme enabled in the app itself?

If so, could we separate them into a different section or something so it's more clear that they aren't intended to follow the same design as the rest of the page?